helm/templates/cluster-role-controller.yaml

```diff
@@ -4,19 +4,11 @@ kind: ClusterRole
 metadata:
   creationTimestamp: null
   name: ack-lambda-controller
-  labels:
-  {{- range $key, $value := .Values.role.labels }}
-    {{ $key }}: {{ $value | quote }}
-  {{- end }}
 {{ else }}
 kind: Role
 metadata:
   creationTimestamp: null
   name: ack-lambda-controller
-  labels:
-  {{- range $key, $value := .Values.role.labels }}
-    {{ $key }}: {{ $value | quote }}
-  {{- end }}
```

Comment:
Looks like you're using an outdated code-generator

Reply:
Done, I have updated the code-generator.

Reply:
This code should not be removed. Can you double check that you included changes to `helm/templates` folder?